apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/types.ts

```diff
@@ -2,4 +2,5 @@ export interface Post {
 	id: number;
 	status: string;
 	title: { raw: string; rendered: string };
+	slug?: string;
```

Comment:
We don't need to add the slug to the type because we're not actually requesting the slug when we call `getEntityRecords()`. We're only asking for some fields i.e. `_fields: 'id,status,title'`.

But I think I see where the confusion is. The
```typescript
		const current = {
			id: // ...
			slug: // ...
			status: // ...
			title: // ...
		};
```
in `useNavItems()` still has a slug field, and it's not used now.